### ISSUE TITLE
Root certificate installation issues

### ISSUE THREAD

**Issue by tomjn** [humanmade/altis-local-server#565]
Some members of my team tried to use v13 of local server for the first time and found that the local environment had an invalid certificate authority. I had been prompted with touch ID to install the certificate during setup, **but they did not.**

I know how to manually install a root certificate but I cannot locate the certificate files Local Server is using. `composer server ssl install` does not resolve the problem and only states if `mkcert` is installed and its version, no success or failure information, or debugging data.

These would be helpful to know:
 - `composer server ssl install` should state explicitly if installation succeeded or failed
 - this should be mentioned in the troubleshooting section of the documentation
 - there is no instruction for manually doing this, we have a mix of multiple operating systems, some users are running this inside environments such as linux VMs where the certificate won't be installed on the host environment and requires manual interventon
 - if automated installation fails it should say so, it does not

**Comment by tomjn** [comment]
It appears that `composer server ssl install` does not install the SSL certificate for the server at all, even though that's what the CLI help for that command says.

Instead it installs `mkcert` which is not what it describes. I found this out by digging through the `readme.md`.

Instead I had to run `composer server ssl generate` with a domain name ( should it not be able to read this from the `composer.json`? )
